<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotPointUI</class>
 <widget class="QDialog" name="RotPointUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Rotation Point</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rotation Point</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="objectPoint">
        <property name="text">
         <string>Object origin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="worldPoint">
        <property name="text">
         <string>World origin (0, 0, 0)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="customPoint">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Custom Point</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="customX">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="customY">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="customZ">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Z:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RotPointUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RotPointUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customPoint</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>worldPoint</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectPoint</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
